models.py

```diff
@@ -1109,15 +1114,15 @@ def __init__(self, *args, **kwargs):
 class FeatureForm(forms.Form):
 
   SHIPPED_HELP_TXT = ('First milestone to ship with this '
-                      'status. Applies to: Enabled by default, Behind a flag, '
-                      'Origin trial, Browser Intervention, and Deprecated. If '
+                      'status. Applies to: Enabled by default, In developer trial, '
+                      'Browser Intervention, and Deprecated. If '
```

Comment:
How are we dealing with removals?

Reply:
For a deprecation:
+ At the "Dev trial for deprecation" stage we prompt for all milestones other than webview
https://screenshot.googleplex.com/C8qBt8JbujayiCw
+ At the "Prepare for deprecation trail" stage we ask for OT start and end milestones
https://screenshot.googleplex.com/AkiauAUbLE56Q8k
+ At the "Prepare to ship" and "Remove code" stages we prompt for all "shipped" milestones.
https://screenshot.googleplex.com/5ZdArhYDDkbfHto
https://screenshot.googleplex.com/JxTPRamWUwYGVQv

Does that sound OK?